index.html

```diff
@@ -1,657 +1,1295 @@
+                                <div id="container-6ede0e2991" class="cmp-container">
+                                <div class="image aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                                  <div data-cmp-lazythreshold="0" data-cmp-src="/content/leaderimpact/lang/en2/_jcr_content/content-parsys/container_1381991251/container/container/Copy%20of%20Copy%20of%20container_copy/image.coreimg{.width}.svg/1669067142072/bulb.svg" data-asset="/content/dam/LeaderImpact/site-./assets/bulb.svg" data-title="logo" id="image-423ca939e7" class="cmp-image" itemscope="" itemtype="http://schema.org/ImageObject">
+                                    <img src="./assets/bulb.svg" class="cmp-image__image" itemprop="contentUrl" data-cmp-hook-image="image" alt="logo">
+                                  </div>
+                                </div>
+                                </div>
+                                <div class="container responsivegrid box-title-wrap aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                                  <div id="container-feb5484fc2" class="cmp-container">
+                                    <div class="aem-Grid aem-Grid--12 aem-Grid--default--12 ">
+                                      <div class="title aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                                        <div id="title-0a2b1292c9" class="cmp-title">
+                                          <h4 class="cmp-title__text">All Resources</h4>
+                                        </div>
+                                      </div>
+
+                                      <div class="text aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                                        <div id="text-cd72b55a7f" class="cmp-text">
+                                          <p>Lorem ipsum dolor sit amet, consetetur sadip.</p>
+                                        </div>
+                                      </div>
+                                    </div>
+                                  </div>
+                                </div>
+                              </div>
+                            </div>
+                          </div>
                         </div>
+                      </div>
                     </div>
+                  </div>
                 </div>
+              </div>
             </div>
-        </section>
-        <section class="newsletter-bar">
-            <div class="container">
-                <div class="title">
-                    <div data-cmp-data-layer="" id="" class="title-wrap">
-                        <h3>Join Thousands of Leaders Leaving a Lasting Impact</h3>
-                    </div>
+          </div>
+        </div>
+
+        <div class="container responsivegrid newsletter-bar aem-GridColumn aem-GridColumn--default--12 layout-grid">
+          <div id="news-container" class="cmp-container">

```

Comment:
Since there could be multiple newsletter bars on a page (I see 2 on the home page and 1 on the resource page already), and in the spirit of reuse of the same component, it would be better if we did not force an id like this and instead targeted `.newsletter-bar .cmp-container`

Reply:
I've differentiate the ID's on the homepage... as news-container and news2-container.